src/utils/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from './asyncUtils';
 export { ObserverClass } from './classObserver';
 export type { ObserverIdentifier, ObserverInterface } from './classObserver';
+export * from './color';
```

Comment:
I think it's a bit difficult to know what is in our repos.
It would be great if we could export it, so that we can use intellisense and dot notation to easier find out:
utils.color.stringToColor

Reply:
Good point!

Updated the export as:
export * as Color from './color'.

Now it's usable like this: (kept the capital letter for 'Color', for sake of consistency)
![image](https://user-images.githubusercontent.com/13872908/134465043-f52022e2-fe96-410a-bbbb-758d8f145292.png)

Reply:
Great :) We should also do the same with the old code in a future PR